layouts/partials/sidebar.html

```diff
@@ -117,6 +117,24 @@ <h2 class="mt-4 taxonomy" id="{{ $key }}-section">{{ T $key }}</h2>
     </div>
     {{- end }}
     {{- end }}
+
+    <h2 class="mt-4">Meetups</h2>
+    <ul class="flex-column">
+      {{- range first 5 (site.Taxonomies.tags.Get "meetup") }}
+      <li>
+        <a href="{{ .RelPermalink }}" class="nav-link" title="{{ .Title }}">
+          
+          {{ if .Params.meetup_date }}
+          {{ $meetupTime := .Params.meetup_date | time }}
+          {{ if lt time.Now $meetupTime }}<b>[Upcoming]</b>{{end}}
+          {{ end}}
+          {{ .Title | markdownify }} {{ if .Params.meetup_date }} ({{ time.Format "Jan 2006" .Params.meetup_date }}){{ end }}
+        </a>
+      </li>
+      {{- end }}
+      <li><a href='{{ ref . "/tags/meetup" }}'>All meetups...</a></li>
+    </ul>
```

Comment:
I don't think this should be inside the list, could we move to a paragraph? Also suggesting a wording change

```suggestion
    </ul>
     <p><a href='{{ ref . "/tags/meetup" }}'>Browse all meetups.</a></p>
```